### ISSUE TITLE
Template props false negative?

### ISSUE THREAD

**Issue by josephzidell** [vuejs/eslint-plugin-vue#1632]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 7.32.0 
- **eslint-plugin-vue version:** 7.18.0
- **Node version:** v14.16.1
- **Operating System:** Mac

**Please show your full configuration:**

```json5
module.exports = {
	root: true,
	env: {
		node: true,
	},
	plugins: [
		'@typescript-eslint',
		'eslint-plugin-tsdoc',
		'jsdoc',
		'vue',
	],
	extends: [
		'eslint:recommended',
		'plugin:vue/base',
		'plugin:vue/essential',
		'plugin:vue/recommended',
		'@vue/standard',
		'@vue/typescript',
		'@vue/eslint-config-typescript/recommended',
		'plugin:chai-friendly/recommended',
		'plugin:promise/recommended',
		'plugin:@typescript-eslint/recommended-requiring-type-checking',
	],
	parser: 'vue-eslint-parser',
	parserOptions: {
		extraFileExtensions: ['.vue'],
		parser: '@typescript-eslint/parser',
		project: './tsconfig.json',
		sourceType: 'module',
	},
	rules: {
		// @typescript-eslint replacements: disable original and use the new one
		indent: 'off',
		'@typescript-eslint/indent': ['error', 'tab'],
		'lines-between-class-members': 'off',
		'@typescript-eslint/lines-between-class-members': ['error', 'always', {exceptAfterSingleLine: true}],
		'no-duplicate-imports': 'off',
		'@typescript-eslint/no-duplicate-imports': 'error',
		'no-extra-parens': 'off',
		'@typescript-eslint/no-extra-parens': 'error',
		'no-invalid-this': 'off',
		'@typescript-eslint/no-invalid-this': 'error',
		'no-unused-vars': 'off',
		'@typescript-eslint/no-unused-vars': ['error', {argsIgnorePattern: '^_'}],
		'padding-line-between-statements': 'off',
		'@typescript-eslint/padding-line-between-statements': [
			'error',
			{blankLine: 'always', prev: '*', next: 'return'},
			{blankLine: 'always', prev: '*', next: ['interface', 'type']},
			{blankLine: 'always', prev: ['block-like', 'directive'], next: '*'},
			{blankLine: 'always', prev: 'multiline-expression', next: 'multiline-expression'},
		],
		'space-before-function-paren': 'off',
		'@typescript-eslint/space-before-function-paren': 'error',
		'space-infix-ops': 'off',
		'@typescript-eslint/space-infix-ops': ['error', {int32Hint: false}],
		// regular rules
		'@typescript-eslint/consistent-indexed-object-style': ['error', 'index-signature'],
		'@typescript-eslint/consistent-type-imports': ['error', {prefer: 'no-type-imports'}],
		'@typescript-eslint/explicit-function-return-type': ['error', {
			allowExpressions: false,
			allowTypedFunctionExpressions: false,
			allowHigherOrderFunctions: false,
			allowDirectConstAssertionInArrowFunctions: false,
			allowConciseArrowFunctionExpressionsStartingWithVoid: false,
		}],
		'@typescript-eslint/member-delimiter-style': 'error',
		'@typescript-eslint/member-ordering': 'error',
		'@typescript-eslint/naming-convention': [
			'error',
			{ // most everything
				selector: 'default',
				format: ['camelCase'],
				filter: {
					regex: '(X-[A-Za-z-]+)', // exclude headers
					match: false,
				},
			},
			{ // parameters can have leading underscores
				selector: ['parameter'],
				format: ['camelCase'],
				leadingUnderscore: 'allow',
			},
			{ // public readonly class properties
				selector: 'classProperty',
				modifiers: ['public', 'readonly'],
				format: ['PascalCase'],
			},
			{ // consts
				selector: 'variable',
				modifiers: ['const'],
				format: ['camelCase', 'PascalCase'],
			},
			{ // arrays
				selector: 'variable',
				types: ['array'],
				format: ['camelCase'],
				suffix: ['s'],
			},
			{ // booleans
				selector: 'variable',
				types: ['boolean'],
				format: ['PascalCase'],
				prefix: ['can', 'did', 'has', 'is', 'should', 'was', 'will'],
			},
			{ // ignore destructured variables
				selector: 'variable',
				modifiers: ['destructured'],
				format: null,
			},
			{ // types
				selector: 'typeLike',
				format: ['PascalCase'],
			},
			{ // rule paths
				selector: ['objectLiteralProperty'],
				format: ['snake_case'],
				filter: '[.]',
			},
			{ // permission names
				selector: ['objectLiteralProperty'],
				format: null,
				filter: '[ ]',
			},
			{ // properties
				selector: ['objectLiteralProperty', 'typeProperty'],
				format: ['camelCase', 'PascalCase', 'UPPER_CASE'],
			},
		],
		'@typescript-eslint/no-confusing-non-null-assertion': 'error',
		'@typescript-eslint/no-confusing-void-expression': 'error',
		'@typescript-eslint/no-meaningless-void-operator': 'error',
		'@typescript-eslint/no-unnecessary-type-constraint': 'error',
		'@typescript-eslint/no-unsafe-argument': 'error',
		'@typescript-eslint/prefer-return-this-type': 'error',
		'@typescript-eslint/type-annotation-spacing': 'error',
		'array-bracket-spacing': ['error', 'never'],
		'array-element-newline': ['error', 'consistent'],
		'arrow-spacing': ['error', {
			before: true,
			after: true,
		}],
		'brace-style': ['error', '1tbs'],
		camelcase: 'error',
		'comma-dangle': ['error', 'always-multiline'],
		'comma-spacing': ['error', {
			before: false,
			after: true,
		}],
		curly: 'error',
		'default-case-last': 'error',
		'default-param-last': 'error',
		'eol-last': ['error', 'always'],
		'jsdoc/check-access': 'error', // Recommended
		'jsdoc/check-alignment': 'error', // Recommended
		'jsdoc/check-indentation': 'error',
		'jsdoc/check-line-alignment': 'error',
		'jsdoc/check-param-names': 'error', // Recommended
		'jsdoc/check-property-names': 'error', // Recommended
		'jsdoc/check-syntax': 'error',
		'jsdoc/check-tag-names': 'error', // Recommended
		'jsdoc/check-types': 'error', // Recommended
		'jsdoc/check-values': 'error', // Recommended
		'jsdoc/empty-tags': 'error', // Recommended
		'jsdoc/implements-on-classes': 'error', // Recommended
		'jsdoc/newline-after-description': 'error', // Recommended
		'jsdoc/no-bad-blocks': 'error',
		'jsdoc/no-defaults': 'error',
		'jsdoc/no-types': 'error',
		'jsdoc/no-undefined-types': 'error', // Recommended
		'jsdoc/require-description': 'error',
		'jsdoc/require-hyphen-before-param-description': 'error',
		'jsdoc/require-jsdoc': ['error', {
			require: {
				ClassExpression: true,
				FunctionDeclaration: true,
				MethodDefinition: true,
			},
		}], // Recommended
		'jsdoc/require-param-description': 'error', // Recommended
		'jsdoc/require-param-name': 'error', // Recommended
		'jsdoc/require-param': 'error', // Recommended
		'jsdoc/require-property-description': 'error', // Recommended
		'jsdoc/require-property-name': 'error', // Recommended
		'jsdoc/require-property-type': 'error', // Recommended
		'jsdoc/require-property': 'error', // Recommended
		'jsdoc/require-returns-check': 'error', // Recommended
		'jsdoc/require-returns-description': 'error', // Recommended
		'jsdoc/require-throws': 'error',
		'jsdoc/require-yields-check': 'error', // Recommended
		'jsdoc/require-yields': 'error', // Recommended
		'jsdoc/valid-types': 'error', // Recommended
		'key-spacing': ['error', {
			beforeColon: false,
			afterColon: true,
			mode: 'strict',
		}],
		'new-parens': ['error', 'always'],
		'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off', // eslint-disable-line no-restricted-syntax
		'no-constant-condition': 'error',
		'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off', // eslint-disable-line no-restricted-syntax
		'no-dupe-keys': 'error',
		'no-extend-native': ['error', {exceptions: ['String']}],
		'no-implicit-coercion': ['error', {disallowTemplateShorthand: true}],
		'no-mixed-operators': 'error',
		'no-mixed-spaces-and-tabs': 'error',
		'no-multiple-empty-lines': ['error', {max: 1}],
		'no-nonoctal-decimal-escape': 'error',
		'no-promise-executor-return': 'error',
		'no-restricted-syntax': [
			'error',
			{
				selector: "CallExpression[callee.property.name='$nextTick'][arguments.length>0]",
				message: 'Usage of $nextTick() with a callback found. Please use await instead',
			},
			{
				selector: "MemberExpression[object.object.name='process'][object.property.name='env'][property.name!='']",
				message: 'Usage of process.env.FOO found. Please use env.get(\'FOO\') instead',
			},
		],
		'no-tabs': 'off',
		'no-trailing-spaces': 'error',
		'no-unreachable-loop': 'error',
		'no-unsafe-optional-chaining': 'error',
		'no-use-before-define': 'off',
		'no-void': ['error', {allowAsStatement: true}],
		'object-curly-newline': ['error', {
			multiline: true,
			consistent: true,
		}],
		'object-curly-spacing': ['error', 'never'],
		'object-property-newline': ['error', {allowAllPropertiesOnSameLine: true}],
		'padded-blocks': ['error', 'never'],
		'prefer-template': 'error',
		'promise/catch-or-return': ['error', {allowFinally: true}],
		quotes: ['error', 'single', {avoidEscape: true}],
		semi: ['error', 'always'],
		'sort-imports': 'error',
		'space-in-parens': ['error', 'never'],
		'space-unary-ops': 'error',
		'template-curly-spacing': ['error', 'never'],
		'use-isnan': 'error',
		'valid-typeof': 'error',

		'vue/array-bracket-newline': ['error', 'consistent'],
		'vue/array-bracket-spacing': ['error', 'never'],
		'vue/arrow-spacing': ['error', {
			before: true,
			after: true,
		}],
		'vue/block-lang': ['error', {
			script: {
				lang: 'ts',
			},
		}],
		'vue/block-tag-newline': 'error',
		'vue/brace-style': ['error', '1tbs'],
		'vue/camelcase': 'error',
		'vue/comma-dangle': ['error', 'always-multiline'],
		'vue/comma-spacing': ['error', {
			before: false,
			after: true,
		}],
		'vue/custom-event-name-casing': ['error', 'kebab-case'], // TODO: change to 'camelCase' when we upgrade to vue 3
		'vue/dot-notation': 'error',
		'vue/eqeqeq': 'error',
		'vue/func-call-spacing': ['error', 'never'],
		'vue/html-button-has-type': 'error',
		'vue/html-indent': ['error', 'tab'],
		'vue/html-self-closing': ['error', {
			html: {
				void: 'always',
				normal: 'never',
				component: 'never',
			},
		}],
		'vue/key-spacing': ['error', {
			beforeColon: false,
			afterColon: true,
			mode: 'strict',
		}],
		'vue/match-component-file-name': 'error',
		'vue/max-attributes-per-line': 'off',
		'vue/mustache-interpolation-spacing': ['error', 'always'],
		'vue/new-line-between-multi-line-property': 'error',
		'vue/next-tick-style': ['error', 'promise'],
		'vue/no-constant-condition': 'error',
		'vue/no-deprecated-v-is': 'error',
		'vue/no-invalid-model-keys': 'error',
		'vue/no-sparse-arrays': 'error',
		'vue/no-template-target-blank': 'error',
		'vue/no-this-in-before-route-enter': 'error',
		'vue/no-unused-refs': 'error',
		'vue/no-use-computed-property-like-method': 'error',
		'vue/no-useless-concat': 'error',
		'vue/no-useless-mustaches': 'error',
		'vue/no-v-text': 'error',
		'vue/object-curly-newline': ['error', {
			multiline: true,
			consistent: true,
		}],
		'vue/object-curly-spacing': ['error', 'never'],
		'vue/object-property-newline': ['error', {allowAllPropertiesOnSameLine: true}],
		'vue/padding-line-between-blocks': ['error', 'always'],
		'vue/prefer-template': 'error',
		'vue/require-emit-validator': 'error',
		'vue/script-indent': ['error', 'tab', {
			switchCase: 1,
		}],
		'vue/singleline-html-element-content-newline': 'off',
		'vue/space-in-parens': ['error', 'never'],
		'vue/space-infix-ops': 'error',
		'vue/space-unary-ops': 'error',
		'vue/template-curly-spacing': ['error', 'never'],
		'vue/v-on-event-hyphenation': ['error', 'always'],
		'vue/valid-next-tick': 'error',
	},
	overrides: [
		{
			files: ['*.spec.ts'],
			rules: {
				// chai-friendly replacements: disable original and use the new one
				'no-unused-expressions': 'off',
				'chai-friendly/no-unused-expressions': 'error',
				// ignore these type-specific check in tests since wrapper.vm doesn't know about the component's methods
				'@typescript-eslint/no-unsafe-assignment': 'off',
				'@typescript-eslint/no-unsafe-call': 'off',
				'@typescript-eslint/no-unsafe-member-access': 'off',
			},
		},
	],
	reportUnusedDisableDirectives: true,
	settings: {
		jsdoc: {
			mode: 'typescript',
		},
	},
};
```

**What did you do?**	

```vue
<template>
  <div
    :data-valid="[one, two]"
    :data-invalid="ones"
  >
    {{ twos }}
  </div>
</template>

<script setup>
import {ref} from "vue";
const one = ref("highlight");
const two = ref("2");
</script>
```

**What did you expect to happen?**
I expected an error that `ones` isn't defined. Also expected an error that `twos` is not defined.

There are no errors that `ones` and `twos` are not defined and are indeed completely invalid.

**What actually happened?**

[13:1]: Combine this with the previous 'const' statement. ( one-var )

**Repository to reproduce this issue**
https://ota-meshi.github.io/eslint-plugin-vue-demo/#eJyVWtuO3DgO/ZVCP+0CYwSYx2xmv6RfVLaqSolsVSS5q2uC/PseUnfZ7ulF0EiZpHWheDmk/OtlNJN8+fryzcv5roWX/31dTqdvk3o7fR21cO6v1xezSPf6wozT6devk38Yd/r9mwW/QBKcb1+q9/HoRqvu/uSkX++gqPlurD/9svLy+3SxZj69vryt8vXlP6/LaBbnT5jj9NcJ/H+9vtzU9abx519f/p0FMGkW+DMwvn0J02CClz9e7Kqle/n6iwb+Mpp5losfJmXl6NUbtvjnH8z57t6H1Uk3vAnrMnUxg7DWPIbLukAeMw5qGR7Cj7dGxj2XkTgY/756OQ13a+7SeiWbsSZ5x7yCBCbhxWDO37EMkKFSK2j8I2npvDVP/NLqIsfnqOWRpNEYC2LOy0VaN4i7+ljUjVjq5LTxHwnLN+jtI4GL0l7aI2ZUn9BDMogDyZuf9SC15GNSR9pTi1aL/Kex6BAcCBexaj/42+FwrIJBeG/VeT0cjlT0OaHPjfc2nNWCN2A7hxJmGRZBhjrMZlIXdajhILrOZ2mz6NF2QYGlLhd1HX7IJ3l6K7li8WBsiW84GScHdek4WtHAZcPNm9liB3GBhQziIZSvBebVY4vLNZxXzbnDF4kh4YPNxuHvcLqBfEwsU8tx0r5hk/0GOOZkk4A7raNfbXM27iYs3iQnRtyhMAEnbQTUJAd5wX68+4S7J/OktbT0d4+ZBHbuvBhvzSLWBWGoWoTbYfYxCkScjboMD+Vv+HVptcWEZjVkLONN6UZ1HNZ2D8nKnysiZqWZypUSk3Qw+OedbSvot1pGkrJymTC+xWnYZcONUciVeLsR8eZ6hS2phQ/D4/ydaiJnEgy7rlWfOUKrKccFWnclQuvaO9vnjoyclXdhOFFvNuSfaHLbrBKOC0cg38VYtByWlY/JGtPzQsTYEEdtxI8NtXfVmr4h1haTaBSKN8SdAStbSCREILl9tzqkQhq3i7nbLc3dzGNL1DsqIu9iYmWGg0P4Cjsko4jpP67nKr1vTTKndJz9rKBG+S7HNZ0wuKMAsUrbLjMIG+Flp655KLIkuPuwyOvwt7Sm0BH9e1EseY8m4HMkX0w9cIAKNFZ+le+JOMnzer1KW5619GTzhUKhXNirawhh5bOkDNJyakPapIcmAZBaEhVm7J8IMtDQSOrl8VvmXZDil4aYH947NeBUrRjOxmgpFprJtxwHZ0yUi9Da36xZr7dMQkDpRrxqcwYgaYkBkvbEBVBq97zVEsyOj+CeqRbPK4Grx00hHd7FmPUCe8IOJkprB8qZ1TsBCXoJpgQb8eKcmYt8ADHMZ6MTBSAS6tA6i5jRi8yFBXvDUfm8AqGpsnSkOt6QLATsIc6baE7qS6cMt+cuSJ0TYLKlxKpGCpuLmKthKJOT0VlR7IYA2XCWcEhZ+SdnOcTm8gDFSh5yRrhWBPwKD1kUWjzriuTERQKNAmvqZ0eFB4ragWI61eIsi/r6HBvyK4oIBzUn4F8RpRvFPc9PGZh/p2TzVDKmWQr7yi0iRsEQ7XEypoFTIzCBmQuGozzRB1QGTdDFXcuDfLGb9VsuD0lxYCWsvZtpWmya4d1we95vcmkLlgo1SWifAxUbAfkqrD0LMsJHymJkd4YD/JCezDof7KEQmeZmJMkOcnUtFYuvN8W0n6upoSnT2L7jPJkzi/cKyw6wzKFZXLbDtlqhcEz/93tJKA+LguPcjWbFbXaTDrX2wIQ7YyxARMNuYVgIW+SSDEJ3y4x87MExMwsnVKE42hnqtvImY7i9Q0vWfACaOgRYFk+DaPl5XSW7809dE0ldHSkUWg0xrQMBAwlJ+YDPtvYLn7JTVUkVhZAhdUyqYMxBrZmdMJSG9upaF2sQFA+bkGUH7TkUgrWZ5KzN+AMBqjqKQMJKN8or+0CZ3umm8Ph4q7n6o44BiAv+YweGHj1O5QasTifeOW5ssxyd8q7Qrncnoc6fZy5Vyp7IiHmtRf/Ik2x2Z+kfUi7Rt5i0QfRIMTh7BbW2SsMhngm5UVJbrvsnRDIRlETnqFlNhd71H5q6GWPfpFUAemMzdoRRead8+o3AOwMWLK2uOmqJhFFCwO/r4mzDoRkV6toMR1IhbuiIlNDVSsydwxhO39SiuQBvLc51MgkmbPZT8QjVHDUNarFuSUdT1YZ9ILLpQFQ8R12KcT/cVmJdjUbRO7wYe1ZsYfttAo8IIumIxdKoJEWHiJWQBSAsl67dRmDoSm0/+0H7ADiLzIVaclJQ/2OvvbDfzYhMKy+b3sOmVoaf/ZAoJvzNTJ0wY6aUEvuRmLnVYFXUcZYSE+Pnxr/ZkLbtglDqRS/ZZgokiJ36vUokxm3zCwfFTRCJ7tdFKkfO2fhcNIdQb7FrdBknADf4qsLqYPs7yTAE5z0YxuzUPWEP6pAdozPZpcaG1TpBYOUIucGJqBiL/zPAPwR1LbeP96HR3lNDtttQMYbsMxxUqXMFGnPeLIZJEATpiP2AkdoMOCGnY/ZWv0RcjG+J8if9y49ca3Lw6qeh1tsO7QED2NAJhWrZ3AgctAKaVFEX1TlFUIFMPYg2Nh6WqTvB77l48d5h0q6k69wYa4TuCvwJdx3javVzYxkNs1dEZObYsnmZusDU5CHqGRVhMVIk34u026TNW6WQ2ikl0S/qfaidINDXRdhnQy8YarNy6iFR0qJZtDG5N0BnaEq9ujkbxptQn7lQH2xUrqpX+3ZUV4R3qyFIwMAl8atyuVlSrI5D9qQcfxMo4Kr34ome4bisUJlASoqIUP+MmLbeEUdjUBcjShZEP+xNxUo6+D85Bo1ULz56Od9E5VZV6khREnGcZ/IVDiUcLd9VDL+kU77w8vWgrAX+laoWbtvp1DqqqQ0BpyHmSq6PA5sYUPn/FdmZNrGz/esq4OYU21V4jTycNxmLzFyKke1oaXOHi1RW2iN1X7M1gEm9tc1Abt21MiFKpAyRqT+HZdW5ayTfSgcJFko1driGas025+lM4VZKIl20CRc7E+x4LiOqOdZno5F2rFaRGaE15xo6X0JWg0RUm22CaD5EguxITWOHy7gKJjAJFSvpIhFmcYUdh3u00pDry/JC83kuKjjqkej5YcX9Xg0EWv5pFu7TJd10baTAa2nBLOHCTTuOG3zpoYOyFYhjJt9btK8nYsbXFNoDkoH7ZAq3SUIfu9B+rhQIqn4eJXDNh5DfXJfYyZqCsjdN7AgrAbWweFf3VkvXrRjmTnKpqK2lvxmVrfMh7MIt11BPhjlidiAMRsXGnfDwwA5cs1PQtZLTEN9FNu+Hvmncd+BYMan3NkJmBSfKuii6eq1bx9RzJPzmY2wm+xmUugSdP80UbiPDGfNParTtNqbZ9Ouuf2UanXdVLaLUfB1o3IaC1DpV6k61SOBU0X0P+x3jvsBJTaH6nS3u6zFfi/dwfCietfo7FD/lkDcgcAsAe/C3LWda6ZxscuyRRpd8sQ//mMqVAzcvNmRXHstKGiCfe7WNorIIY4iGxZB+edK3IOkZgPIaG9TUEcj97Bx5AwpvkVRV0tBnMlUftceze1iWq7TUx0E9Vbw/z1LtmGi50xnfqhip2tveVlEyneQ9bq4Gz/yb3+YkW2e+zMpP+J1aDwRSCoOjb3mkKo6ttusPt7xAyu3ipIWy3cIj0A54GXwMuQMGnX/X8DCcL0/KOC0uNuOGgLG6G3hqUin/UOV6jrpualnzc+xONK4TcyZByXwdEy6nEuTu0mGbWnJnKUCOomfOhFcRejddOoj6b7QRkmkoA3La06ujv53wVtUqlJWqy7NuVAAW0v21S+0U7yywKIW3EjaYsUg5bVdXqqcUFpvmAFJ9jGbJLoazmZ6tLxyWR4el0UdlESEdxP46M7RmGgXC71Q/hePLHndQV1H3JTXuXKZs+jGdE8RcSR2VhVuJvKY0RS0S9+XwKELK4YBTtSWqAES3vo1WAiH7V74Vbvsw/JRv+Jqjq/bV0Ns4W/HbKnKvgtyrHpnWRjj3UKGrrbvboapBeG0YCUKczVzwQikAQixnWysaCcRqxdu+y5W+HOSTxyHavqqlL8dWvpfjNxO5tLPDFfqeZ4Yu3B6ne6fAvJiJ0+1lCwH7GFdwYL4LIByY7Dwa1g0z3dJnY9HoghJS1K85PE1NKB+PJaUkHInkjO20SDBjROfrFBLJlZHvtSr4rSDTGjl3/CuVhS8CaGkM3VNQ+aA9wdfS2/N9W+V3F4tXdUZ09M8v6YkjufmujgXpK8uqUbsVWEYony+N4q3PsahVou1E7otYUzp2+xJ10zsizeMxR023QPFj15t4C1/LhefDlxAf5vwxDm2uVMF74rf4+cdn1KAuliAjX7Mp/8FW+W5Z8DfNQ9yvGy5mXI+XHUoEGrr6+msjhdoISuS1invdctxKGioM/k/dEWRhQv2Bwq7U6s3H++EGCHyTtEAfjf7TWVOZDYNEij9WK0epCYiAuq5ka8fDEZf1FAu8afiEDQOdELoG7jQRD3CP5ffv/wGLJpQ5

**Comment by ota-meshi** [comment]
Thank you for posting issue.

This plugin has no rules to check it. If you want a check, we need to create a new rule.

**Comment by ota-meshi** [comment]
If you need a new rule, could you open an issue that proposes the rule?
